### PR TITLE
WIP : activity graph

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: components/features/graphs/activity-graph/utils/isCurrentWeek.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { isSameWeek } from "date-fns";
+
+export function isCurrentWeek(date: Date): boolean {
```
Comment (haydencleary) [inline components/features/graphs/activity-graph/utils/isCurrentWeek.ts:3]: https://date-fns.org/v3.6.0/docs/isThisWeek ?

File: app/migration/u/[githubLogin]/features/activity-graph/activity-graph.tsx [hunk 2]
```diff
@@ -1,7 +1,59 @@
...
+  }
+
+  const mockTooltipContent = "5 contributions • 2 rewards";
+  const mockIcon: TIcon.Props = { remixName: "ri-medal-2-fill" };
+
+  return (
+    <div className="flex w-full flex-col gap-4">
+      <div className="flex w-full flex-row items-center justify-between gap-1">
+        <Typography variant="title-m" className="flex-1">
+          Activity
```
Comment (haydencleary) [inline app/migration/u/[githubLogin]/features/activity-graph/activity-graph.tsx:25]: Translation

File: components/features/graphs/activity-graph/use-activity-graph.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+          id: getWeekId(week),
+          startDate: startWeek,
+          endDate: endWeek,
+          days: eachDay.map(day => ({ date: day, id: getDayId(day) })),
+        };
+      })
+    );
+  }, [end, start]);
+
+  const splitedWeeks = useMemo(() => {
```
Comment (haydencleary) [inline components/features/graphs/activity-graph/use-activity-graph.ts:47]:
```suggestion
  const splitWeeks = useMemo(() => {
```

File: components/features/graphs/activity-graph/components/week/week.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+export function Week({ week, data }: TWeek.Props) {
+  const weekNumber = useMemo(() => getWeekNumber(week.startDate), [week]);
+
+  const tooltipContent = useMemo(
+    () => (
+      <div className="flex flex-col gap-1">
+        <p>
+          <Translate token={"v2.features.activityGraph.week"} /> {weekNumber}
+        </p>
+        {data?.tooltipContent ? <p>{data?.tooltipContent}</p> : null}
```
Comment (haydencleary) [inline components/features/graphs/activity-graph/components/week/week.tsx:21]:
```suggestion
        {data?.tooltipContent ? <p>{data.tooltipContent}</p> : null}
```

**Review by haydencleary — CHANGES_REQUESTED** [review]

File: components/features/graphs/activity-graph/utils/isCurrentWeek.ts (hunk 1 above)
Comment (alexbeno) [reply components/features/graphs/activity-graph/utils/isCurrentWeek.ts:3]: yes would be better, just deleted it because it is not used

**Review by haydencleary — APPROVED** [review]

**Pull request merged**